src/Models/ColorModel.vala

```diff
@@ -21,72 +21,74 @@
 
 /*
  * Model to keep track of Fills and Borders colors of an item. We use this
- * model to easily bind the GtkListBox UI to the Fills and Borders Components.
+ * model to easily bind the ListBox UI to the Fills and Borders Components.
  */
 public class Akira.Models.ColorModel : GLib.Object {
-    private unowned Lib.Components.Fill? fill;
-    private unowned Lib.Components.Border? border;
+    public signal void value_changed ();
 
-    public Type type;
-    public enum Type {
-        FILL,
-        BORDER
-    }
+    public class SignalBlocker {
```

Comment:
Why does this signal blocker exist?

Reply:
It's used by the FillitemModel when the node is updated for the first time, which blocks the `on_value_changed()` in the subclass.
Should the signal blocker live only in the subclass?